### ISSUE TITLE
Inconsistent enemy text

### ISSUE THREAD

**Issue by RSBat** [nroutasuo/level13#183]
**Game version**
0.3.2 (alpha)

**Browser**
Firefox 86.0.1 (64-bit)

**Description**
World seed 8629, level 10 11E 14N. In the description "Passage to the north is blocked by a group of pests" but fight button is "Fight bots". Actual fight is with pests

**Comment by nroutasuo** [comment]
Do you remember / can you still check if the last of the 3 fights is with bots?
It's a bit confusing though how the possible enemies are presented here, I should definitely improve it.

**Comment by RSBat** [comment]
No, already cleared them. Same thing happened on same seed, level 10 16E 19N, but already cleared them too

**Comment by RSBat** [comment]
Happened again on level 9 1E 10N. First two fights were with ghost bats, last - security bot

**Comment by nroutasuo** [comment]
Thanks for the details!
